libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/customConfiguration.ts

```diff
@@ -669,6 +669,13 @@ function getLabelsConfiguration(chartOptions: IChartOptions, _config: any, chart
                     verticalAlign: "middle",
                 },
             },
+            waterfall: {
+                dataLabels: {
+                    ...DEFAULT_LABELS_CONFIG,
+                    verticalAlign: "top",
```

Comment:
It's just a hardcode property, so my point is that it would be better to move to a waterfall configuration waterfallConfiguration.
Please check! 

Also for `y: -20` 

Thank you!

Reply:
I have moved them to the `waterfallConfiguration.ts`. Thank you for your suggestion!